src/main/java/BulkInsertApp.java

```diff
@@ -0,0 +1,69 @@
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Map;
+
+import org.yaml.snakeyaml.Yaml;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+public class BulkInsertApp {
+
+    public static final int USER_ID = 1;
+    public static final String APPLICATION_YML = "/application.yml";
+
+    @Parameter(names = {"--count"}, description = "Number of records to insert", required = true)
+    private int count;
+
+    public static void main(String[] args) {
+        BulkInsertApp app = new BulkInsertApp();
+        JCommander.newBuilder().addObject(app).build().parse(args);
+        app.run();
+    }
+
+    private void run() {
+        String insertSQL = "INSERT INTO boards"  + " (title, content, user_id) VALUES (?, ?, ?)";
+
+        Map<String, Object> config = yamlConfigReader();
+        Map<String, String> dataSource = (Map<String, String>) config.get("datasource");
+        final String url = dataSource.get("url");
+        final String user = dataSource.get("user");
+        final String password = String.valueOf(dataSource.get("password"));
+
+        try (Connection connection = DriverManager.getConnection(url, user, password)) {
```

Comment:
What is this try-resource syntax? Can you explain me what's the difference with just try-catch ?

Reply:
`try-with-resources` can resources automatically closed. But, `try-catch` can't. 
So manually close resources with null checking.  
As a result, we can make the code more concise and prevent mistakes